ldj-readme-renderer/README_template.md

```diff
@@ -0,0 +1,66 @@
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=Ali-chakaroun_ISEP-LivingDocumentation&metric=coverage)](https://sonarcloud.io/summary/new_code?id=Ali-chakaroun_ISEP-LivingDocumentation)
+[![Technical Debt](https://sonarcloud.io/api/project_badges/measure?project=Ali-chakaroun_ISEP-LivingDocumentation&metric=sqale_index)](https://sonarcloud.io/summary/new_code?id=Ali-chakaroun_ISEP-LivingDocumentation)
+[![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=Ali-chakaroun_ISEP-LivingDocumentation&metric=reliability_rating)](https://sonarcloud.io/summary/new_code?id=Ali-chakaroun_ISEP-LivingDocumentation)
+[![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=Ali-chakaroun_ISEP-LivingDocumentation&metric=sqale_rating)](https://sonarcloud.io/summary/new_code?id=Ali-chakaroun_ISEP-LivingDocumentation)
+# Living Documentation Java
+
+This project is an extension of the [Living Documentation][ldoc] set of tools to
+Java. As part of the Living Documentation toolchain, it can be used to generate
+documentation and diagrams from source code so that documentation is always
+up-to-date.
+
+## State of the project
+
+As of right now, this project is linked to a student assignment originating from the University of
+Twente and InfoSupport. The aim of the project is to make the [LivingDocumentation][ldoc] project
+capable to handle multilingual projects. One aspect of this is to make LivingDocumentation
+available for the Java environment; which is the purpose of this repository.
+
+Within the aspect of making LivingDocumentation available for the Java environment, two features
+can be identified (which are copied from [LivingDocumentation][ldoc] description):
+
+* [**Analyzer**](#analyzer): A tool to analyze Java projects.
+* [**Libraries**](#support-libraries): Assists in generating applications that can create plain text files such as MarkDown, AsciiDoc, PlantUML, Mermaid, and more.
+
+> **Note**: As this repository is currently association to a University assignment, it is unfortunately not possible to directly contribute to this project. 
+> Furthermore, some of the described functionalities may not yet be finalised/are subject to change.
+
+## Getting started
+
+The process of rendering documentation can be summarized as follows:
+1. **Analysis**: Run the Analyzer over the source code of your project. This creates a LivingDocumentation JSON file.
```

Comment:
The only disadvantage with that is that if the options change (either the literal spelling or usage) this would have to be changed manually as opposed to the table below.
Would be kind of undermining if as a consequence of this the readme would contain a contradiction somewhere in the future (the word undermining is maybe a bit extreme but you get the sentiment).

This is why I tried to keep the 'fixed parts' rather general. Curious how you see this

Reply:
I get what you mean. Indeed, let's keep it like this.